### PR TITLE
New Logo Kaniko

### PR DESCRIPTION
**Pull request by ggcarlosr**
kaniko-header

### CODE REVIEW

**Comment by priyawadhwa** [comment]
Cool, I think you just need to do two more things:
* Add the image to the "Logo" file you created
* Sign the CLA above!

File: Logo [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (ggcarlosr) [inline Logo]: kaniko-header

**Comment by priyawadhwa** [comment]
So we need the actual image uploaded, rather than just a comment -- why don't we try this instead:

In your [fork](https://github.com/ggcarlosr/kaniko) click the `Upload Files` button, and upload the logo image. 

Then, at the bottom, you want to select the option to `Create a new branch for this commit and start a pull request` and then click `Commit changes` to create another pull request

**Comment by ggcarlosr** [comment]
Ready my brother

**Comment by ggcarlosr** [comment]
list the pull request please confirm that it has been done correctly

**Comment by AkihiroSuda** [comment]
This logo looks like 大 (Chinese: da, Japanese: dai, Korean: dae), which means "big" or "great" :+1: , although rotated :smile:

**Comment by dlorenc** [comment]
@googlebot I signed it!

**Comment by dlorenc** [comment]
I've manually verified the CLA was signed, so I'm not sure what's up with the check. I'll go ahead and merge. Thank you so much @carlossg!

**Pull request merged**